packages/express/README.md

```diff
@@ -0,0 +1,153 @@
+# @loopback/express
+
+This package adds middleware support for LoopBack 4 and allows Express
+middleware to be plugged into LoopBack seamlessly. It's used by `@loopback/rest`
+to support Express middleware with `InvokeMiddleware` action within the REST
+sequence.
```

Comment:
Is it worth the effort to maintain this feature as a standalone package, when it's not an optional extension and it's always used by (included in) `@loopback/rest`?

Reply:
I think so. My plan is to allow the usage of an existing Express application without dragging in `@loopback/rest`.